core-bundle/src/DependencyInjection/ContaoCoreExtension.php

```diff
@@ -176,6 +181,21 @@ static function (ChildDefinition $definition, AsPickerProvider $attribute): void
         );
     }
 
+    public function configureFilesystem(FilesystemConfig $config): void
+    {
+        $filesStorageName = 'files';
+
+        $config
+            ->mountLocalAdapter('%kernel.project_dir%/files', 'files')
```

Comment:
Don't we need `contao.upload_path` here somewhere?

Reply:
Yes, already changed locally. :smile:

Reply:
see 4dedecd301365d938bea37d068722fdd6edea19f